### ISSUE TITLE
WaveRNN bad result even trained 1000k steps.

### ISSUE THREAD

**Issue by tsungruihon** [fatchord/WaveRNN#177]
I have tried to train fatchord wavernn on aIndonesia dataset. But after training on like 1000k steps, the result still worse. May i ask for some advice ? @fatchord  

The log is below.

```
| Epoch: 1/3846 (520/520) | Loss: 4.2473 | 4.2 steps/s | Step: 1001k |
| Epoch: 2/3846 (520/520) | Loss: 4.2304 | 4.1 steps/s | Step: 1001k |
| Epoch: 3/3846 (520/520) | Loss: 4.2351 | 4.1 steps/s | Step: 1002k |
| Epoch: 4/3846 (520/520) | Loss: 4.2877 | 4.0 steps/s | Step: 1002k |
| Epoch: 5/3846 (116/520) | Loss: 4.1097 | 4.1 steps/s | Step: 1002k |
```

The sample is below.

[indonesia_mol.wavernn.zip](https://github.com/fatchord/WaveRNN/files/4352561/indonesia_mol.wavernn.zip)

**Comment by Ivona221** [comment]
I am also training a model on Macedonian dataset. Even after 400K steps it is still really bad. The loss is around 4.9. Should i keep training or modify some parameters and try again?

My sample is below:
[output10.wav.zip](https://github.com/fatchord/WaveRNN/files/4461712/output10.wav.zip)

**Comment by 1zxLi** [comment]
I have the same problem as you. Has your result improved?

**Comment by xuexidi** [comment]
I met the same problem.....who can help me!

**Comment by anjanakethineni** [comment]
can somebody tell me the process to get pre-trained models for other datasets?

**Comment by najmuzsakib11** [comment]
@fatchord I have the same problem. Loss is around 5.0 after 1000K. Can you give some advice how to improve loss? Should I train more?

**Comment by CrazyCharles6** [comment]
The effect of WaveRNN trained with GTA is too bad by LJSpeech-1.1 dataset.

**Comment by magungh1** [comment]
@tsungruihon hello there, currently I also train wavernn for indonesian language and getting the same result with yours, mind if you share the tips to reduce the loss ??
